pycqed/instrument_drivers/physical_instruments/ZurichInstruments/ZI_base_instrument.py

```diff
@@ -5,6 +5,7 @@
 import matplotlib.pyplot as plt
 import logging
 import re
+from datetime import datetime
```

Comment:
Again, we should not allow this PR to include any conflicting changes with the ZI Base Instrument driver!